src/components/Footer/Footer.scss

```diff
@@ -43,7 +43,7 @@ input[type="email"] {
   padding: 128px 49px;
   font-family: "Mulish", sans-serif;
 }
-span {
+.subscribe {
```

Comment:
Please use SCSS nesting, otherwise we can't use this class .subscribe in other places
See https://github.com/pesto-students/little-tags-debajyoti-xyz/commit/edfceac2f5cf10b5e9242af2f19d98659951ddb4